shopfloor/services/single_pack_transfer.py

```diff
@@ -123,16 +123,21 @@ def start(self, barcode, confirmation=False):
         package_level = package_level.filtered(
             lambda pl: pl.state not in ("cancel", "done")
         )
-        if not package_level:
-            if self.work.menu.allow_move_create:
-                package_level = self._create_package_level(package)
+        message = self.msg_store.no_pending_operation_for_pack(package)
+        if not package_level and self.work.menu.allow_move_create:
```

Comment:
I put the two conditions on one line to avoid flake8 lint regarding the complexity of the method.